src/components/Auth/MfaSetupModal.tsx

```
@@ -1,497 +1,107 @@
-  const supabase = createClient();
-  const [step, setStep] = useState<"choice" | "qr" | "verify" | "recovery" | "complete">("choice");
-  const [method, setMethod] = useState<"totp" | "sms">("totp");
-  const [verificationCode, setVerificationCode] = useState("");
-  const [phoneNumber, setPhoneNumber] = useState("");
-  const [copiedKey, setCopiedKey] = useState(false);
-  const [copiedCodes, setCopiedCodes] = useState(false);
-  const [savedRecoveryCodes, setSavedRecoveryCodes] = useState(false);
-  const [loading, setLoading] = useState(false);
-  const [errorMsg, setErrorMsg] = useState("");
+export const MfaSetupModal: React.FC<MfaSetupModalProps> = ({ isOpen, onClose }) => {
+    const [mfaData] = useState(generateMfaSecret());
```

ESLint

[error] Delete `··`

(prettier/prettier)